x-pack/test/functional/services/observability/alerts/pagination.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+const ALERTS_ROWS_PER_PAGE_SELECTOR = 'tablePaginationPopoverButton';
```

Comment:
Nit: This one is suffixed with SELECTOR, but the others are not. 

Nit: These constants aren't exported, and they live in a folder called `alerts` so maybe it's redundant to prefix them with ALERTS?

Reply:
I will delete the SELECTOR for this one and yep I can remove ALERTS from all of them since it is indeed redundant